### ISSUE TITLE
Add 192000 to Pulseaudio SUPPORTED_SAMPLERATES

### ISSUE THREAD

**Issue by swwa** [thestk/rtaudio#274]
The following change works nice on Debian Buster and Ubuntu 20.04. Please consider adding the change to RtAudio.

```
 #include <pulse/simple.h>
 #include <cstdio>
 
-static const unsigned int SUPPORTED_SAMPLERATES[] = { 8000, 16000, 22050, 32000,
-                                                      44100, 48000, 96000, 0};
+static const unsigned int SUPPORTED_SAMPLERATES[] = {
+  8000, 16000, 22050, 32000, 44100, 48000, 88200, 96000, 192000, 0
+};
 
 struct rtaudio_pa_format_mapping_t {
   RtAudioFormat rtaudio_format;
```

**Comment by ic-scm** [comment]
This entire suppored sample rate restriction seems pointless, see this issue: https://github.com/thestk/rtaudio/issues/272

**Comment by sonoro1234** [comment]
Yes, but the fix in #272 is still not applied

**Comment by swwa** [comment]
@radarsat1 Thanks for the fix, it helps my project!
